x-pack/legacy/plugins/lens/public/plugin.tsx

```diff
@@ -97,6 +97,7 @@ export class LensPlugin {
       data,
       editorFrame: editorFrameSetupInterface,
       formatFactory,
+      executeTriggerActions: npStart.plugins.uiActions.executeTriggerActions,
```

Comment:
This is basically tech debt already, but I don't see an alternative that exists.

Reply:
don't use npStart here, pass your dependencies into setup method.
once you are in new platform the second parameter to `setup()` wil lbe all the plugin contracts.
currently you already get the `expressions`, `data` and `embeddable` there ... add `uiActions` (between line 86 and 87) there as well